TombEngine/Renderer/RendererSprites.cpp

```diff
@@ -414,11 +419,11 @@ namespace TEN::Renderer
 			_shaders.Bind(Shader::InstancedSprites);
 
 			// Set up vertex buffer and parameters.
-			UINT stride = sizeof(Vertex);
-			UINT offset = 0;
+			unsigned int stride = sizeof(Vertex);
```

Comment:
I am not sure if it's a good idea to do that. This data type is derived from the arguments of `GetAddressOf`.